src/rgw/rgw_aws_sns.h

```diff
@@ -0,0 +1,25 @@
+
+//
```

Comment:
i don't think we put copyright notices in the code :-)
this is tracked in git, when you sign the commits